src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -211,6 +224,12 @@ public ITaskItem[] ResponseFiles
             get { return (ITaskItem[])_store[nameof(ResponseFiles)]; }
         }
 
+        public bool SkipCompilerExecution
```

Comment:
I suppose we could have just one bool property that is true to emit command line and false to execute the command line instead as that would fit my requirement, but it seems unnecessarily coupled. What if folks want to both compile and get the command line? That's why I separated the possibility of skipping.
There are plenty of ways to play practical jokes on folks. I'm not too concerned with this providing yet another way.